app/src/lib/editors/win32.ts

```diff
@@ -24,6 +24,7 @@ function getRegistryKeys(editor: ExternalEditor): ReadonlyArray<string> {
       return [
         // 64-bit version of VSCode - not available from home page but just made available
         'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{EA457B21-F73E-494C-ACAB-524FDE069978}_is1',
+        'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{1287CAD5-7C8D-410D-88B9-0D1EE4A83FF2}_is1',
```

Comment:
This only covers the 64-bit version (I think) - are we able to find out what the install location is for the 32-bit version so we cover both cases?

Reply:
Do we need to cover the 32-bit version since Desktop only runs on 64-bit Windows?

Reply:
VSCode was 32-bit only up until a couple of months ago, so I think lots of people are still running the 32-bit version.

Reply:
The 32-bit VS Code Insider key is `HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{C26E74D1-022E-4238-8B9D-1E7564A36CC9}_is1`

Reply:
@shiftkey Added